external-crates/move/crates/move-vm-runtime/src/execution/interpreter/mod.rs

```diff
@@ -75,3 +87,5 @@ macro_rules! set_err_info {
 }
 
 pub(crate) use set_err_info;
+
+use super::tracing::tracer::VMTracer;
```

Comment:
Seems a weird place for this `use` (and the above one too). Move to top of the file?